tests/credential_plugins_test.py

```diff
@@ -137,13 +137,19 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
-        from awx_plugins.credentials.dsv import SecretsVault  # noqa
+    def test_dsv_import(self) -> None:
+        from awx_plugins.credentials.dsv import SecretsVault  # noqa: F401
```

Comment:
@djyasin so the best action here is to try to actually fix the underlying problem. I wonder if it's okay for us to do a `pytest.importorskip()` (if this test is even needed / cc @chrismeyersfsu).

If that's not acceptable for some reason, let's keep it as it is in your current patch.